bot.py

```diff
@@ -152,7 +152,7 @@ async def on_message(message):
 
     # context sensitive command - scheudle
     if '!schedule' in message.content:
-        response = 'Check out our stream schedule here: https://www.twitch.tv/jennytree95/schedule'
+        response = 'Check out our stream schedule here: https://www.twitch.tv/' + twitch_userID + '/schedule'
```

Comment:
Can we use a format string here instead? Just looks a bit cleaner.

```suggestion
        response = f'Check out our stream schedule here: https://www.twitch.tv/{twitch_userID}/schedule'
```

Reply:
Yarp job done, like this style. I will make it my standard.